packages/redux/src/reducers/defaultWorkspaceSettings.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * The Server Configs Values retrieved from the Server
```

Comment:
A little pedantic, but at this layer we should just say these are the default settings. In our particular implementation, some of them are read in from the server, and some of them fallback to the hardcoded defaults we have in the Web UI. 
```suggestion
 * Default values for workspace settings. Used when the user has not modified a setting yet.
```